daheinze-oscheer/integers/adder_test.go

```diff
@@ -0,0 +1,21 @@
+package integers
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestAdd(t *testing.T) {
+	sum := Add(2, 2)
+	expected := 4
+
+	if sum != expected {
+		t.Errorf("expected %d but got %d", expected, sum)
+	}
+}
```

Comment:
This should include a few more test cases. I know it's just integers for now and just an addition, but it makes sense to adopt the test cases approach already. Let's discuss this in our session today.